data/remnant/remnant missions.txt

```diff
@@ -2208,3 +2208,420 @@ mission "Remnant: Face to Maw 2B"
+		description "	The cloaking technology is built into the Starling's hull; it cannot be removed or transferred to another ship."
+
+
+
+mission "Remnant: Crystal Research 5"
+	name "Scan the <npc> in <waypoints>"
+	description `Go scan the <npc> to record the Keystone post-wormhole.`
+	landing
+	waypoint "Peragenor"
+	source "Viminal"
+	destination "Viminal"
+	npc "scan outfits" "save"
+		government "Remnant"
+		personality coward disables plunders staying surveillance target mining
+		system "Peragenor"
+		ship "Starling (Keystone)" "Winds of Light"
+	to offer
+		has "Remnant: Crystal Research 4: done"
+	on offer
+		conversation
+			`You exit the ship to find that the pilot is actually Aeolus, who has already marshaled some dock workers with a number of wagons and lifters to unload the cargo.`
+			`	"Ah, <first>, thank you for retrieving all these Keystones." he pauses to stare at a crate bearing the image of a bunch of smiling Hai, then shakes his head and turns back to you.`
+			`	"Our researchers have decided that we need to test how a ship performs with one of these Keystones compared to a regular Key Stone. My ship has been outfitted with a keystone built from the samples you brought us earlier. We request that you meet us in Peragenor, where you will scan us after going through the wormhole. Then we will return here and swap out the Quantum Keystone built with Hai crystals for a Quantum Key Stone made from our crystals. Then we will go back to Peragenor and perform another scan. By using the same ships for both tests, we should be able to get the best data."`
+			choice
+				`	"Wait, you are a pilot?"`
+				`	"Okay."`
+					accept
+			`	"Oh, yes. It is actually quite common among the Remnant. I am not a particularly good pilot, but it is considered an important skill that is taught to almost everyone as a part of a general education. It is especially worthwhile for those of us who work directly with starships on a daily basis and may be called upon to move or fly one at a moment's notice." His gestures suggest both pride and humility as he talks about his piloting skills. "Anyway, let's go!"`
+				accept
+	on visit
+		dialog `You land on <planet>, but you haven't scanned the Starling in <waypoints>. Make sure you scan the ship before returning.`
+	on complete
+		payment 150000
+
+
+
+mission "Remnant: Crystal Research 6"
+	name "Scan the <npc> in <waypoints>"
+	description `Go scan the <npc> to record the Key Stone post-wormhole.`
+	landing
+	waypoint "Peragenor"
+	source "Viminal"
+	destination "Viminal"
+	npc "scan outfits" "save"
+		government "Remnant"
+		personality coward disables plunders staying surveillance target mining
```

Comment:
```suggestion
		personality staying target uninterested
```

Reply:
I gave it those personalities so that it would act effectively like other Remnant - except that it would run away if it got too badly damaged. This is not the same as uninterested, which is fairly bland and basically just translates "ignores the player."